domains/authorization/Authorization.API/API/Authorization/Controllers/ApiModels.cs

```diff
@@ -39,6 +39,6 @@ public enum ClientType
 public record CreateClientRequest(Guid IdpClientId, string Name, ClientType ClientType, string RedicrectUrl);
 public record CreateClientResponse(Guid Id, Guid IdpClientId, string Name, ClientType ClientType, string RedirectUrl);
 
-public record UserOrganizationConsentsResponseItem(string ClientName, long ConsentDate);
+public record UserOrganizationConsentsResponseItem(Guid ClientId, Guid OrganizationId, string ClientName, long ConsentDate);
```

Comment:
OrganizationId should no be part of this request. Use org_cvr claim and IdpClientId to find consent.
Controller method used OrgCvr policy, so the org_cvr claim exists.